### PR TITLE
Reformat date

### PR DESCRIPTION
**Pull request by jasmyn2244**
____ Wrote Tests ____ Implemented __x__ Reviewed

## Neccesary checkmarks:

    [x] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor

## Description of Change:

    description: reformatted the dates on the single sightings page
    

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

### CODE REVIEW

File: src/Components/SingleSighting.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ const SingleSighting = () => {
     }
   }
 
+  const formatDate = (date) => {
```
Comment (krdurrett) [inline src/Components/SingleSighting.js:65]: This a great helper function!

**Review by krdurrett — APPROVED** [review]
Thanks Jasmyn!

File: src/Components/SingleSighting.js (hunk 1 above)
Comment (GraceGardner) [inline src/Components/SingleSighting.js:65]: Sweet! Good idea to format these dates

**Review by GraceGardner — APPROVED** [review]
Good work here!

**Pull request merged**